Assets/HoloToolkit-Examples/Utilities/Scripts/SwapVolume.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule.Tests
```

Comment:
The namespace here should probably be HoloToolkit.Unity.Examples.Utilities

Reply:
Should this live in the Solvers folder?

Reply:
>Should this live in the Solvers folder?

Nah, it's only for the example.

Reply:
`HoloToolkit.Unity.Examples.Utilities` isn't currently in use, and most of the others in the folder are in the `HoloToolkit.Unity.InputModule.Tests` namespace. We probably need to take a namespace pass in the examples.

Reply:
fair enough

Reply:
I'll update the namespace here though.